### DISCUSSION TITLE
How to add scroll event on modal?

### DISCUSSION THREAD

**Discussion by fuyunliu** [fomantic/Fomantic-UI#858] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
# scroll event on modal

Mac OS 10.14.5
Chrome 75.0.3770.100 (64bit)
Fomantic UI 2.7.6

## Problem
```javascript
function ping() {alert('pong')}

// not working
window.addEventListener('scroll', ping)

// not working
document.addEventListener('scroll', ping)

// not working
document.querySelector('div.ui.modal').addEventListener('scroll', ping)

```

**Comment by lubber-de** [comment, marked as answer]
@fuyunliu It depends on the kind of scrollable modal:
- A normal modal gets scrollable via the `page dimmer`, thus you set your listener to the dom node `.page.dimmer`
- A modal having `scrolling content` gets scrollable via the `.content`, thus you set your listener to the dom node `.modal .content`


See difference here:
https://jsfiddle.net/f2mradto/1/
